man/plotting-functions.Rd

```diff
@@ -168,17 +176,23 @@ switchPlotfun(method)
   \code{HaploPainter} is a perl script/tool for pedigree plotting bundled in
   the package that requires however some dependencies that might not be
   present on every system. Thus, the package checks on startup whether
-  all requirements for \code{HaploPainter} are available.
+  all requirements for \code{HaploPainter} are available. This check can
+  be skipped by using \code{check=FALSE} when calling \code{switchPlotFun}.
+  While using this argument is generally not recommended, it is of use
+  when only writing \code{HaploPainter} input files, which does not make
+  use of the \code{HaploPainter} plotting backend.
 
   If \code{HaploPainter} is used, the plot can only be exported to a pdf
   or png device, while, if \code{kinship2} is used, the plot can also be
   directly plotted and displayed (if \code{device="plot"} is specified).
 
   \code{HaploPainter} plotting supports also \code{device = "txt"} in
```

Comment:
Option `device = "txt"` is described here. Isn't that enough?

Reply:
I would also mention the option `"txt"` where the `device` parameter is described. I don't have the documentation in front of me right now but there should be somewhere an `\item{device}` or similar.

Reply:
Got it. Done with the last commit.